pudb/debugger.py

```diff
@@ -262,24 +262,22 @@ def set_trace(self, frame=None, as_breakpoint=None, paused=True):
         # See pudb issue #52. If this works well enough we should upstream to
         # stdlib bdb.py.
         # self.reset()
-
-        if paused:
```

Comment:
FYI the only change here is removing this line (this is clearer if you look at https://github.com/inducer/pudb/pull/688/files?w=1)